src/ocm/components/clusterConfiguration/review/ReviewOperatorsTable.tsx

```diff
@@ -26,7 +27,8 @@ export const ReviewOperatorsTable = ({ cluster }: { cluster: Cluster }) => {
           operator,
         }),
       )
-      .map((operator) => ({
+      .map((operator, index) => ({
+        rowId: `operator-${operator}-${index}`,
```

Comment:
Will we have duplicate operator ids? If not then using the index seems obsolete.

Reply:
Ahh, it did in my tests for LVM(S) as I mocked the data and acted as if all existing operators where active.
In reality that situation cannot happen, as only one of LVM / LVMS would be active at the same time.

I'll remove the index.